omod/src/main/java/org/openmrs/module/coreapps/page/controller/MarkPatientDeadPageController.java

```diff
@@ -0,0 +1,85 @@
+
+package org.openmrs.module.coreapps.page.controller;
+
+import org.apache.commons.logging.Log;
+import org.apache.commons.logging.LogFactory;
+import org.openmrs.Concept;
+import org.openmrs.ConceptAnswer;
+import org.openmrs.Patient;
+import org.openmrs.api.PatientService;
+import org.openmrs.api.context.Context;
+import org.openmrs.module.appui.UiSessionContext;
+import org.openmrs.ui.framework.SimpleObject;
+import org.openmrs.ui.framework.UiUtils;
+import org.openmrs.ui.framework.annotation.SpringBean;
+import org.openmrs.ui.framework.page.PageModel;
+import org.springframework.web.bind.annotation.RequestParam;
+
+import java.util.Collection;
+import java.util.Date;
+
+
+/**
+ * Marking a patient as dead
+ */
+
+public class MarkPatientDeadPageController {
+    protected final Log log = LogFactory.getLog(this.getClass());
+
+    public void controller(UiSessionContext sessionContext, PageModel model) {
+    }
+
+    public void get(@SpringBean PageModel pageModel, @RequestParam(value = "breadcrumbOverride", required = false) String breadcrumbOverride, @SpringBean("patientService") PatientService patientService, @RequestParam("patientId") String patientId) {
+        String conceptId = Context.getAdministrationService().getGlobalProperty("concept.causeOfDeath");
```

Comment:
The Concept Cause of death in the global properties is usually Concept not uuid. if It was  I would have used uuid